backend/routes/paymentRoutes.js

```diff
@@ -2,13 +2,36 @@ import express from 'express';
 
 import { protect } from '../middleware/authMiddleware.js';
 import { config, order, validate } from '../controllers/paymentController.js';
+import validateRequest from '../middleware/validator.js';
+import {body, check} from 'express-validator';
 
 const router = express.Router();
 
+const validator = {
+    order: [
+        check('body').exists().withMessage('Request body is required')
```

Comment:
Please fix the order validation, it checks for the existence of a property named 'body' in the request body. And there is no such property available in the request body.
```
        const options = req.body;
        console.log(options)  
//  {
//  amount: 8278850,
//    currency: "INR",
//   receipt: "receipt#65ad8fa293b9b1db8df33365"
// }
        
```

Reply:
fixed